tests/FormTest/packages.config

```diff
@@ -1,10 +1,16 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<packages>
-  <package id="Autofac" version="4.6.2" targetFramework="net461" />
-  <package id="Microsoft.AspNet.WebApi.Client" version="5.2.4" targetFramework="net461" />
-  <package id="Microsoft.AspNet.WebApi.Core" version="5.2.4" targetFramework="net461" />
-  <package id="Microsoft.Net.Compilers" version="1.2.1" targetFramework="net46" developmentDependency="true" />
-  <package id="Microsoft.Rest.ClientRuntime" version="2.3.2" targetFramework="net46" />
-  <package id="Microsoft.WindowsAzure.ConfigurationManager" version="3.2.1" targetFramework="net46" />
-  <package id="Newtonsoft.Json" version="10.0.3" targetFramework="net461" />
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Autofac" version="4.2.0" targetFramework="net461" />
+  <package id="Microsoft.AspNet.WebApi.Client" version="5.2.4" targetFramework="net461" />
+  <package id="Microsoft.AspNet.WebApi.Core" version="5.2.4" targetFramework="net461" />
+  <package id="Microsoft.Net.Compilers" version="1.2.1" targetFramework="net46" developmentDependency="true" />
+  <package id="Microsoft.Rest.ClientRuntime" version="2.3.10" targetFramework="net461" />
+  <package id="Microsoft.WindowsAzure.ConfigurationManager" version="3.2.1" targetFramework="net46" />
+  <package id="Newtonsoft.Json" version="10.0.3" targetFramework="net461" />
+  <package id="System.AppContext" version="4.3.0" targetFramework="net461" />
+  <package id="System.Collections.Immutable" version="1.4.0" targetFramework="net461" />
+  <package id="System.Diagnostics.StackTrace" version="4.3.0" targetFramework="net461" />
+  <package id="System.IO.FileSystem" version="4.3.0" targetFramework="net461" />
+  <package id="System.IO.FileSystem.Primitives" version="4.3.0" targetFramework="net461" />
+  <package id="System.Reflection.Metadata" version="1.5.0" targetFramework="net461" />

```

Comment:
There are many new packages here. Is this intentional?

Reply:
This is intentional because the Tests\FormTest project is referencing them so we want them to be restored by NuGet.

Reply:
I've just reviewed the Tests\FormTest project and we can get rid of the referenced NuGet's thus these packages won't be necessary.